### ISSUE TITLE
Transformer Training

### ISSUE THREAD

**Issue by thisisashukla** [thisisashukla/inclusive-ai#13]
**Sub Tasks**

- [ ] Update data loader function to create a ID matrix for each batch
- [ ] Code training loop using Teacher forcing
- [ ] Loss function with BLUE score
- [ ] Create visualization to track training

**Success Criteria**:

- Working function
